src/Microsoft.SqlTools.Hosting.v2/Utility/CommandOptions.cs

```diff
@@ -37,7 +46,12 @@ public CommandOptions(string[] args, string serviceName)
                         switch (argName)
                         {
                             case "-enable-logging":
-                                EnableLogging = true;
+                                break; //ignore this old option for now for backward compat with older opsstudio code - to be removed in a future checkin
```

Comment:
Yes we will doing to changes to vscode-mssql code and azure data studio code to start using the new options of -tracing-level instead of -enable-logging in future checkin.